db/sqlglue.c

```diff
@@ -7329,6 +7330,7 @@ static int rootpcompare(const void *p1, const void *p2)
     return strcmp(tp1->zName, tp2->zName);
 }
 
+int gbl_debug_systable_locks = 0;
```

Comment:
Can we have this enabled for DEBUG builds? This will allow us to automatically test these changes without having to explicitly flip this switch.